### DISCUSSION TITLE
Share Your High Score!

### DISCUSSION THREAD

**Discussion by Lioness100** [Lioness100/guess-that-lang#6] [Q&A] [answered]
# Hi everyone! 👋

Thanks for playing my game! If you'd like to show off your expert skills :), go ahead and share your high score with a screenshot!
The top score will be pinned!

**Comment by Lioness100** [comment]
My high score is 3510, which I'm pretty proud of 🙂 
![image](https://user-images.githubusercontent.com/65814829/184217674-b9c64dd9-4fa5-4f1d-989c-13ee95c1f760.png)

**Comment by Uriopass** [comment]
Got 2100! It's hard because sometimes there's only comments and it could be anything.

![image](https://user-images.githubusercontent.com/5420739/184503845-962decf1-f808-47e8-9d17-10c1b8dfec37.png)

**Reply by Lioness100** [reply]
Aaayy nice score! Yeah comments can be hard, I'm going to work on #9 when I get the chance to remove comments altogether.

**Reply by Lioness100** [reply]
Update: comments will now be removed (along with a bunch of other stuff like extraneous newlines). This is published in the latest version :)

**Reply by tuxiqae** [reply]
At least it's not shell scripts that run a couple of nvm commands hence are tagged as JS :joy:

**Comment by thunderstorm010** [comment]
![image](https://user-images.githubusercontent.com/66248615/184552842-4d1e6c9f-77c8-42ae-9c6c-b83f6a8b8f63.png)

**Reply by Lioness100** [reply]
Nice!! 🧑‍🍳🤌

**Comment by mellowmarshe** [comment]
![image](https://user-images.githubusercontent.com/35433518/184945352-27a8eaf2-ca29-404c-a6d2-4f9722568b68.png)

edit: got 2600 this time

**Reply by Lioness100** [reply]
🎉🎉 congrats!

**Comment by cs-t1** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/52403031/185340571-7a723adc-7624-4efc-b3f2-56394d24066b.png)

I guess I was helped by the heaps of `function startLevel(map) {...` JS blocks I got. I also made the delay before code gets revealed a tad longer as 1.5s is too short IMO. 

Curious about how far I could get with `shuffling` enabled.

**Reply by Lioness100** [reply]
Nice score! What do you suggest the default wait time should be?

**Reply by cs-t1** [reply]
Thanks! idk maybe 2s or 2.5s but this may be too slow for some users

**Reply by Lioness100** [reply]
I'm gonna try to get a higher score now because it's not a good look to be beaten at my own game 😈😈

**Reply by Lioness100** [reply]
![image](https://user-images.githubusercontent.com/65814829/185399322-f6c371c5-7657-437a-832a-ef7a0e10f3cd.png)

I tied... 😫

**Reply by cs-t1** [reply]
Wow probably wouldn't be able to replicate it even if I wanted

**Comment by skeetcha** [comment]
![image](https://github.com/user-attachments/assets/2fae2323-6265-47c4-a488-b6b1c15b69e7)

Apparently my weakness is R (never really used it ever).

**Reply by Lioness100** [reply]
Haha nice score! I'm glad there's someone playing after all this time
